### PR TITLE
Added tests for default environment under cel package

### PR DESCRIPTION
**Pull request by Manoramsharma**
#### What type of PR is this?
/kind feature 

#### What this PR does / why we need it:

Added unit tests for default and environment under `cel` package 
#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by Manoramsharma** [comment]
@wzshiming I observed same check for binary runtime failed with an error message this `Test kwokctl/kwokctl_binary_benchmark failed.` . 

So I changed the `default_test.go` writing units tests by `testify` assert rule and  made a final commit but again got this lint error which says 
```
  Running [/home/runner/golangci-lint-1.58.2-linux-amd64/golangci-lint run  --timeout=30m --config=.golangci.yaml] in [/home/runner/work/kwok/kwok] ...
  Received 3446 of 3446 (100.0%), 0.0 MBs/sec
  level=warning msg="[config_reader] The configuration option `linters.errcheck.ignore` is deprecated, please use `linters.errcheck.exclude-functions`."
  level=error msg="Running error: context loading failed: no go files to analyze: running `go mod tidy` may solve the problem"
```

**Note**: I am working a fork which has latest changes from `upstream/main` . Looking forward for your guidance over this.

Thanks

File: pkg/utils/cel/default_test.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cel
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```
Comment (wzshiming) [inline pkg/utils/cel/default_test.go:22]: This package is currently not expected to be imported

Comment (Manoramsharma) [reply pkg/utils/cel/default_test.go:22]: I will revert the import and write the tests without using the `testify` package.

**Comment by Manoramsharma** [comment]
`Test/ test-kwok-e2e` check is working fine on my local system. Providing you the screenshot of the passing test below:
Screenshot 2024-06-05 at 4 42 12 PM

**Pull request closed without merging**